codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -205,6 +205,9 @@ update these file paths accordingly.
       );
     }
     directoriesVisited.add(directory.absolute.path);
+
+    await cleanupEntitlements(directory);
```

Comment:
does this happen AFTER we've parsed the entitlements file?

Reply:
yeah. This happens in the `visitDirectory` calls, which is after the `processRemoteZip` call where we did the initial parsing of entitlements

Reply:
ahh, ok. Do we only parse entitlements for remote zips, and not local zips?

Reply:
yeah `remoteZip` is the name i had for the engine binaries we downloaded from gcloud. And yes we only process remote zips